gulp/tasks/clean.js

```diff
@@ -1,21 +1,21 @@
+/* eslint-env node */
```

Comment:
Updated to fix this, the folder-level .eslintrc.json now has the right configuration to make this folder pick up the right setting for the `strict` rule.